scripts/utils/goimports_ignore.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find "$(pwd)" -type f -name '*.go' -a ! -name '*zz_generated*' -exec goimports -local github.com/liqotech/liqo -w {} \;
```

Comment:
Would move directly to the makefile, as a one-liner

Reply:
yes you are right, thank you